web_src/js/features/common-global.js

```diff
@@ -56,9 +56,8 @@ export function initGlobalEnterQuickSubmit() {
 }
 
 export function initGlobalButtonClickOnEnter() {
-  $(document).on('keypress', '.ui.button', (e) => {
-    if (e.keyCode === 13 || e.keyCode === 32) { // enter key or space bar
-      if (e.target.nodeName === 'BUTTON') return; // button already handles space&enter correctly
+  $(document).on('keypress', '.ui.button:not(button,a)', (e) => {
+    if (e.code === ' ' || e.code === 'Enter') {
```

Comment:
Good change, we should change all `e.which` and `e.keyCode` to `e.code` as the former two are deprecated.

Reply:
Yep, we should get rid of them (in another PR).
The funny thing is:
While there are a few `e.keyCode` in our code, there is no `e.which` except for the file we got through Fomantic.